### PR TITLE
tsne table loading plus tests (and refactor of checkDB)

### PR DESCRIPTION
**Pull request by pcm32**
Logic, testing and docs for scxa-tsne table loading.

### CODE REVIEW

**Comment by pcm32** [comment]
All tests passing, this is ready to be merged, could you please rev @suhaibMo? Thanks!

File: tests/random-data-set.bats [hunk 1]
```diff
@@ -100,3 +100,32 @@
...
+  run load_db_scxa_tsne.sh
+  echo "output = ${output}"
+  [ "$status" -eq 0 ]
+}
+
+@test "TSNE: Check number of loaded rows" {
+  # Get third line with count of total entries in the database after our load
+  count=$(echo "SELECT COUNT(*) FROM scxa_tsne" | psql $dbConnection | awk 'NR==3')
+  # TODO improve, highly dependent on test files we have, but in a hurry for now.
+  run [ $count -eq 250 ]
```
Comment (suhaibMo) [inline tests/random-data-set.bats:128]: Any particular reason why are we expecting counts 250 ?

**Review by suhaibMo — APPROVED** [review]
Overall looks good. Making a separate script to check Db connection routine loading making more efficient.

File: tests/random-data-set.bats (hunk 1 above)
Comment (pcm32) [reply tests/random-data-set.bats:128]: this is based on the test files I'm using really... not ideal, but I wanted a quick test.

**Comment by pcm32** [comment]
Thanks @suhaibMo!

**Pull request merged**